OperatorComb.java

```diff
@@ -0,0 +1,93 @@
+// Time Complexity : O(4^n) where n is the number of elements in the array and we have 4 decisions at each instance 
+// Space Complexity : O(n) where n is the number of operands in the stringbuilder
```

Comment:
Exponential in case of recursive

Reply:
Oh, this is for backtracking.

Reply:
@super30admin I have a question here:
In Expression Add, We have 0 as a special case "if(index != i and num.charAt(index) == 0) break".
In the code, I couldn't figure out why the 0 is getting ignored/skipped in the case of "105" as input, because it should take 0 in the number into consideration and how the code line above is fixing it?

Reply:
This is ignoring just the preceding zero **for pivot**.. That is ignore processing the pivot at zero and move to next pivot.